pkg/sdk/helper_test.go

```diff
@@ -711,6 +717,56 @@ func createStage(t *testing.T, client *Client, database *Database, schema *Schem
 	}, stageCleanup
 }
 
+func createDynamicTable(t *testing.T, client *Client) (*DynamicTable, func()) {
+	t.Helper()
+	return createDynamicTableWithOptions(t, client, nil, nil, nil, nil)
+}
+
+func createDynamicTableWithOptions(t *testing.T, client *Client, warehouse *Warehouse, database *Database, schema *Schema, table *Table) (*DynamicTable, func()) {
+	t.Helper()
+	var warehouseCleanup func()
+	if warehouse == nil {
+		warehouse, warehouseCleanup = createWarehouse(t, client)
+	}
+	var databaseCleanup func()
+	if database == nil {
+		database, databaseCleanup = createDatabase(t, client)
+	}
+	var schemaCleanup func()
+	if schema == nil {
+		schema, schemaCleanup = createSchema(t, client, database)
+	}
+	var tableCleanup func()
+	if table == nil {
+		table, tableCleanup = createTable(t, client, database, schema)
+	}
+	name := NewSchemaObjectIdentifier(schema.DatabaseName, schema.Name, randomString(t))
+	targetLag := TargetLag{
+		Lagtime: String("2 minutes"),
+	}
+	query := "select id from " + table.ID().FullyQualifiedName()
+	comment := randomComment(t)
+	ctx := context.Background()
+	err := client.DynamicTables.Create(ctx, NewCreateDynamicTableRequest(name, warehouse.ID(), targetLag, query).WithOrReplace(true).WithComment(&comment))
+	require.NoError(t, err)
+	entities, err := client.DynamicTables.Show(ctx, NewShowDynamicTableRequest().WithLike(&Like{Pattern:String(name.Name())}).WithIn(&In{Schema: schema.ID()}))
+	require.NoError(t, err)
+	require.Equal(t, 1, len(entities))
+	return &entities[0], func() {
+		require.NoError(t, client.DynamicTables.Drop(ctx, NewDropDynamicTableRequest(name)))
+		if tableCleanup != nil {
+			tableCleanup()
+		}
+		if schemaCleanup != nil {
+			schemaCleanup()
+		}
+		if databaseCleanup != nil {
+			databaseCleanup()
+		}
+		if warehouseCleanup != nil {
+			warehouseCleanup()
+		}
```

Comment:
Is something wrong with this file? It looks like you cut out closing `}` ?

Reply:
yes you are right haha. fixed.

Reply:
Well, it seems it's still wrong :(

Reply:
apparently i did not push my commit 😮